ci/packaging/suse/skuba_spec_template

```diff
@@ -32,7 +32,7 @@ Source1:        vendor.tar.gz
 Patch0:         0001-Patch-out-load-balancer.patch
 BuildRequires:  go-go-md2man
 BuildRequires:  golang-packaging
-BuildRequires:  golang(API) >= 1.12
+BuildRequires:  golang(API) = 1.13
```

Comment:
Here it needs to have

`BuildRequires:  go >= 1.13`

Otherwise the package build will fail

https://build.suse.de/package/view_file/home:michaljura:branches:Devel:CaaSP:5/skuba/skuba.spec?expand=1

Reply:
I think with very little adaptations we can do 1.14, so it might be worth relaxing the = to a >= ?

Reply:
See discussion [here](https://github.com/SUSE/skuba/pull/997#pullrequestreview-377299149), the build doesn't work with `>=`, I verified that this is still the case now.

We need to pick one, I'd be happy to change it to 1.14?

Reply:
I missed it. =1.13 is probably good enough then. 1.14 will trigger other issues, which I am not sure we are in hurry to have :p

Reply:
@chentex can you revisit your -1? As discussed here, >= 1.13 doesn't work.